migrations/1687269060_addFooterCertificateLinks.ts

```diff
@@ -0,0 +1,105 @@
+import { Client, SimpleSchemaTypes } from "@datocms/cli/lib/cma-client-node";
+
+export default async function (client: Client) {
+  const newFields: Record<string, SimpleSchemaTypes.Field> = {};
+  const newFieldsets: Record<string, SimpleSchemaTypes.Fieldset> = {};
+
+  console.log("Creating new fields/fieldsets");
+
+  console.log('Create fieldset "Certificates" in model "Footer" (`footer`)');
+  newFieldsets["712332"] = await client.fieldsets.create("47595", {
+    title: "Certificates",
+    collapsible: true,
+  });
+
+  console.log(
+    'Create Single-line string field "B corp link" (`b_corp_link`) in model "Footer" (`footer`)'
+  );
+  newFields["9263009"] = await client.fields.create("47595", {
+    label: "B corp link",
+    field_type: "string",
+    api_key: "b_corp_link",
+    validators: { format: { predefined_pattern: "url" } },
+    appearance: {
+      addons: [],
+      editor: "single_line",
+      parameters: { heading: false },
+    },
+    default_value: "",
+    fieldset: newFieldsets["712332"],
+  });
```

Comment:
Should we use the `external_link` block for this? @Frankwarnaar

Reply:
An external link requires a title as well. I think we could indeed use an external link indeed and then use the title for the screen reader text in the link.

Reply:
Would this be a modular content with a single external link record?
I need to match it with the it logo's, or I can add an image field in the external link and make it not required?

Reply:
> Would this be a modular content with a single external link record?

Yes

>  I need to match it with the it the logo's, or I can add an image field in the external link and make it not required?

Think it's best to 'match it with the logo', as in still have two fields specifically for bcorp & dda, both being a modular content field.

Reply:
Could you make sure to make the field labels singular while the api key is plural please? So for instance label the field 'DDA link', while using `dda_links` as api key. Then it looks logical for the content manager, but it also feels logical in the code to pick `ddaLinks[0]`.

Reply:
✅

Reply:
@Frankwarnaar Do we actually want the shortened version of dda or was that an example (kinda agree it's getting long).

Reply:
It was an example. I'm not sure